inspire_json_merger/merger_config.py

```diff
@@ -80,11 +80,22 @@ def factory(config_type):
 
 
 class MergerConfigurationOperations(object):
+    """General Configuration for the Merger
+
+    Attributes:
+        default_dict_merge_op   Default operation for the merge of dicts
+        default_list_merge_op   Default operation for the merge of lists
+        list_dict_ops = None    Dictionary of rules for the merge of dicts
+        list_merge_ops = None   Dictionary of rules for the merge of dicts
+        comparators = None      Dictionary of comparators
+        conflicts = None        List of conflicts that have to be filtered
```

Comment:
Maybe we can improve this docstring using the Napoleon style: `var_name(var_type): <<description>>` :books: